app/test_main.py

```diff
@@ -1,3 +1,25 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize("cat_age, dog_age, expected_result", [
+    (0, 0, [0, 0]),
+    (14, 14, [0, 0]),
+    (15, 15, [1, 1]),
+    (23, 23, [1, 1]),
+    (24, 24, [2, 2]),
+    (27, 27, [2, 2]),
+    (28, 28, [3, 2]),
+    (28, 29, [3, 3]),
+    (100, 100, [21, 17])
+])
+def test_get_human_age(
+        cat_age: int,
+        dog_age: int,
+        expected_result: list
+) -> None:
+    if not isinstance(cat_age, int) or not isinstance(dog_age, int):
+        raise TypeError("Arguments should be int type")
```

Comment:
You should create a new test for incorrect types fot the tested function, not make actions in your test